### PR TITLE
chore: expand user inputs outside of bash and javascript

### PR DESCRIPTION
**Pull request by mbarbero**
While I don't see any issue in current code, it's better practice to place the expression in environment variable before the start of the script.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -80,9 +80,10 @@ jobs:
       - name: Prepare release
         id: prepare-release
         shell: bash
+        env:
+          RELEASE_VERSION: ${{ github.event.pull_request.title }}
```
Comment (netomi) [inline .github/workflows/release.yml:84]: this cant be right. That would set the release version to the title of the PR that triggered the workflow.
However that workflow is not triggered by a pull request.

Comment (mbarbero) [reply .github/workflows/release.yml:84]: my bad, this is old clipboard content that has been pasted 🤦

Comment (mbarbero) [reply .github/workflows/release.yml:84]:
```suggestion
        env:
          RELEASE_VERSION: ${{ github.event.inputs.version }}
```

**Comment by netomi** [comment]
the change LGTM. Something that I learned the hard way is that input variables are not accessible anywhere inside a workflow file as crazy as that sounds (see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#context-availability to get a list of places where each context is available).

So inputs should be available at jobs.<job_id>.steps.env so should be fine.

While looking at change, I also see that this can be further shortened. Instead of `github.event.inputs` you could use `inputs` directly afaict.

**Comment by mbarbero** [comment]
Was following https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable, but if you say that `inputs` are treated differently, then let's just abandon this change.

**Comment by netomi** [comment]
thing seem to have changed, now inputs seem to be available everywhere it seems, so its good.

**Pull request merged**